MobileCenter/MobileCenterTests/MSCustomPropertiesTests.m

```diff
@@ -47,17 +47,31 @@ - (void)testKeyValidate {
   
   // Then
   assertThat([customProperties properties], hasCountOf(0));
-  
+
+  // Too long key.
+  // When
+  NSString *tooLongKey = [@"" stringByPaddingToLength:129 withString: @"a" startingAtIndex:0];
```

Comment:
I think in general we should also test that exact max values (60, 128, 128) pass the validation.